client/jetpack-connect/auth-form-header.jsx

```diff
@@ -48,60 +49,31 @@ export class AuthFormHeader extends Component {
 		return 'logged-in';
 	}
 
-	getPartnerSlug() {
-		const { partnerId } = this.props.authQuery;
-
-		switch ( partnerId ) {
-			case 51945:
-			case 51946:
-				return 'dreamhost';
-			case 49615:
-			case 49640:
-				return 'pressable';
-			case 51652: // Clients used for testing.
-				return 'dreamhost';
-			default:
-				return '';
-		}
-	}
-
-	getHeaderImage() {
-		const partnerSlug = this.getPartnerSlug();
+	getHeaderText() {
+		const { translate } = this.props;
+		const { partnerSlug } = this.props.authQuery;
 
-		let image = false;
+		let host = '';
 		switch ( partnerSlug ) {
 			case 'dreamhost':
-				image = '/calypso/images/jetpack/dreamhost-jetpack-logo-group.png';
+				host = 'Dreamhost';
 				break;
 			case 'pressable':
-				image = '/calypso/images/jetpack/pressable-jetpack-logo-group.png';
+				host = 'Pressable';
+				break;
+			case 'milesweb':
+				host = 'Milesweb';
+				break;
+			case 'bluehost':
+				host = 'Bluehost';
 				break;
 		}
 
-		if ( ! image ) {
-			return null;
-		}
-
-		return (
-			<div className="jetpack-connect__auth-form-header-image">
-				<img width={ 128 } height={ 42.5 } src={ image } />
-			</div>
-		);
-	}
-
-	getHeaderText() {
-		const { translate } = this.props;
-		const partnerSlug = this.getPartnerSlug();
-
-		if ( partnerSlug ) {
-			switch ( partnerSlug ) {
-				case 'dreamhost':
-					return translate( 'In partnership with Dreamhost' );
-				case 'pressable':
-					return translate( 'In partnership with Pressable' );
-				default:
-					return translate( 'Completing set up' );
-			}
+		if ( host ) {
+			return translate( 'Jetpack, in partnership with %(host)s', {
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?